File: app/src/main/java/com/escom/calmind/model/TestResult.java
package com.escom.calmind.model;

public class TestResult {

    private final StressResult stressResult;
    private final ResilienceResult resilienceResult;
    private final TraumaResult traumaResult;

    public TestResult(StressResult stressResult, ResilienceResult resilienceResult, TraumaResult traumaResult) {
        this.stressResult = stressResult;
        this.resilienceResult = resilienceResult;
        this.traumaResult = traumaResult;
    }

    public TraumaResult getTraumaResult() {
        return traumaResult;
    }

    public ResilienceResult getResilienceResult() {
        return resilienceResult;
    }

    public StressResult getStressResult() {
        return stressResult;
    }
}

File: app/src/main/java/com/escom/calmind/model/TestResult.kt
package com.escom.calmind.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import kotlinx.serialization.Serializable

@Serializable
@Parcelize
data class TestResult(
    val stressResult: StressResult,
    val resilienceResult: ResilienceResult,
    val traumaResult: TraumaResult
) : Parcelable
